source/js/iframeAcceptance.js

```diff
@@ -1,9 +1,9 @@
 class IframeAcceptance {
     initIframes() {
         
-        function markup(height) {
-            return `<div data-iframe-container style="height: ${height}; width:100%">
-            <div class="u-level-top u-position--absolute u-align-middle u-padding__x--3 u-padding__y--3 u-flex-direction--column u-overflow--auto" data-suppressed-iframe style="width:100%;height:${height};backdrop-filter:blur(30px);display:flex;">
+        function markup(height, classes) {
+            return `<div data-iframe-container style="height: ${height}; width:100%;" class="${classes}">
+            <div class="u-level-2 u-position--absolute u-align-middle u-padding__x--3 u-padding__y--3 u-flex-direction--column u-overflow--auto" data-suppressed-iframe style="width:100%;height:${height};backdrop-filter:blur(30px);display:flex;">
```

Comment:
html not indented correctly hard to see what closing tag belongs to which opening tag etc